api/tests/unit/domain/services/pole-emploi-service_test.js

```diff
@@ -26,21 +37,32 @@ describe('Unit | Service | Pole Emploi Service', function() {
             referenceExterne: 123456,
             dateDebut: '2020-09-01T00:00:00.000Z',
             elementsEvalues: [] } } };
-      const host = 'https://url-externe';
+
       // when
-      const generatedLink = poleEmploiService.generateLink(sending, host);
+      const generatedLink = poleEmploiService.generateLink(sending);
 
       // then
-      expect(generatedLink).to.equal('https://url-externe/api/pole-emploi/envois?curseur=eyJpZEVudm9pIjo0NTYsImRhdGVFbnZvaSI6IjIwMjEtMDUtMDFUMDA6MDA6MDAuMDAwWiJ9');
+      expect(generatedLink).to.equal('https://url-externe/pole-emploi/envois?curseur=eyJpZEVudm9pIjo0NTYsImRhdGVFbnZvaSI6IjIwMjEtMDUtMDFUMDA6MDA6MDAuMDAwWiJ9');
     });
   });
 
   describe('#decodeCursor', function() {
-    it('should decode the string', function() {
-      const str = 'eyJpZEVudm9pIjo0NTYsImRhdGVFbnZvaSI6IjIwMjEtMDUtMDFUMDA6MDA6MDAuMDAwWiJ9';
-      const decodedData = poleEmploiService.decodeCursor(str);
+    context('when the parameter is a string', ()=> {
```

Comment:
**suggestion (non-blocking) :** `when there is a cursor`
Est-ce qu'il y a pas des validations à avoir vu que c'est un paramètre qui vient d'une requête